libcxx/include/__exception/win32_exception_ptr.h

```diff
@@ -0,0 +1,430 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I think the new header and source files should be added to `libcxx/include/CMakeLists.txt` and `libcxx/src/CMakeLists.txt`.